upgrade/capsule.py

```diff
@@ -135,6 +136,14 @@ def satellite_capsule_upgrade(cap_host, sat_host):
 
     if settings.upgrade.foreman_maintain_capsule_upgrade:
         foreman_maintain_package_update()
+        # capsule certs regeneration required for 7.0 ystream upgrade
+        if settings.upgrade.from_version == '6.10':
+            execute(generate_capsule_certs, cap_host, True, host=sat_host)
+            scp_opts = "-o 'StrictHostKeyChecking no"
+            execute(
+                lambda: run(f"scp {scp_opts} {cap_host}-certs.tar root@{cap_host}:/root"),
+                host=sat_host
+            )
         upgrade_using_foreman_maintain(sat_host=False)
```

Comment:
Add this function in the task.py file and call it in both z-stream and y-stream capsule upgrade.
```suggestion
        def capsule_cert_update(cap_host, sat_host):
    """
    Use to generate the capsule certificate on the satellite and upload it on
    the capsule
    """
    execute(
        generate_capsule_certs,
        cap_host,
        True,
        host=sat_host
    )
    execute(lambda: run(f"scp -o 'StrictHostKeyChecking no' {cap_host}-certs.tar "
                        f"root@{cap_host}:/home/"), host=sat_host)
```

Reply:
I considered to create a new task function, thanks for helping me to realize it is actually a good direction.

For z-stream upgrade it is not required.

Reply:
Updated.